docker/aws/deploy_lambda.py

```diff
@@ -0,0 +1,17 @@
+import sys
+from sessions import get_session
+
+# Create IAM client
```

Comment:
Is this comment still relevant, now that the session code is in a separate file? Maybe I'm misunderstanding what you mean by "IAM client".

Reply:
No, it's no longer relevant, I've taken it out.

Reply:
Thanks. :+1:

Same goes for update_service.py and invoke_lambda.py. Sorry, I forgot to mention that in my first comment!

Reply:
Ah, I should have checked those. I've removed the comments from them as well.